amoro-format-iceberg/src/main/java/org/apache/amoro/table/TableProperties.java

```diff
@@ -115,6 +115,9 @@ private TableProperties() {}
       "self-optimizing.full.rewrite-all-files";
   public static final boolean SELF_OPTIMIZING_FULL_REWRITE_ALL_FILES_DEFAULT = true;
 
+  public static final String SELF_OPTIMIZING_FILTER = "self-optimizing.filter";
+  public static final String SELF_OPTIMIZING_FILTER_DEFAULT = null;
```

Comment:
If it is set to null here, is it possible that NPE may occur during subsequent verification? Can we use **""** instead?

Reply:
<img width="532" alt="image" src="https://github.com/user-attachments/assets/68f85e64-ea79-4016-b749-0d2bdaba968b" />

It won't cause a NullPointerException (NPE) when used only on this side. Moreover, I noticed that in the documentation (configurations.md), many parameter default values are NULL. I haven't seen any default values set as "". I wonder how it would be represented in the documentation if the default value were "".